### ISSUE TITLE
Cauldron Documentation: Content does not reflow to fit horizontally within the viewpoint

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#904]
Original auditor issue: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/6e304616-da82-11ec-a684-83cd68d07db6?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=3&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

URL: https://cauldron.dequelabs.com/components/Table

## Description
Content does not reflow to fit horizontally within the viewpoint when page width is adjusted to 320px.

Element 1: Column header "Email" and the data related to it in the table is not visible properly at reflow.
Location: Below- Table example-  basic

Element 2: Column headers "Email " and the data related to it in the table  is not visible at reflow
Location: Below the "Border variant Table".

Element 3: Column headers "Email & Actions" and the data related to it in the table  is not visible at reflow
Location: Below the "sortable table".

Element 4: Column headers "Email" and the data related to it in the table  is not visible at reflow
Location: Below the "Border variant Table with Table Footer".

Note: For reflow, resize the browser viewport to 640px width by 512px height and zoom in to 200%
### RULE : 
The content MUST be viewable without scrolling horizontally when the viewport is set to 320 CSS pixels wide.

### HOW TO FIX: 
Fix this issue by using ANY of the following techniques:
1. Use CSS techniques such as media queries, grids, or flexbox to reflow content down to 320px width.
2. Provide options within the content to switch to a layout that does not require the user to scroll horizontally to read a line of text.

### REFERENCE: 
Deque University:https://dequeuniversity.com/class/responsive-zoom2/zoom/linear-text-reflow
https://dequeuniversity.com/class/responsive-zoom2/responsive/
WCAG Understanding document:
https://www.w3.org/WAI/WCAG21/Understanding/reflow

### BACKGROUND: 
People with low vision often read content on a desktop computer by scaling content using the browser zoom - up to 400%. When zooming content creates the need to scroll in the direction of reading in order to reveal lines that are cut off by the viewport, it significantly increases the effort required to read text and sometimes causes content hidden off-screen with no way to scroll or pan to it. When content is created so that it reflows at screen widths down to 320px, users can view a web page in a 1280px width browser screen and zoom the content to 400% (320px) without having to scroll side to side to read and without losing access to content.

![overflow issue on tables](https://user-images.githubusercontent.com/1062039/219796754-d15f456c-f53d-4ff7-ab7e-c32ccc83bd4e.png)

**Comment by scurker** [comment]
This isn't an issue with cauldron persay, but rather an issue with the documentation itself. The "official" way to do this would be to put the data into a DescriptionList at whatever media query breakpoint you need. But we should fix this somehow within documentation to not be an accessibility issue for this particular table.
